test/curve_analysis/test_baseclass.py

```diff
@@ -226,12 +231,43 @@ def test_end_to_end_single_function(self):
         result = analysis.run(test_data)
         self.assertExperimentDone(result)
 
+        curve_data = result.artifacts("curve_data").data
+        np.testing.assert_array_equal(curve_data.series_name, "test")
```

Comment:
I guess this returns array. Does assert_array_equal broadcast second value?

Reply:
It broadcasts scalars and raises an assertion if the two objects are not equal: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html
I just used it since we already use it in this manner in tests.